app/controllers/api/v1/games_controller.rb

```diff
@@ -1,15 +1,10 @@
 class Api::V1::GamesController < ApplicationController
-  before_action :param_validation
-
   def show
     game = RawgFacade.get_game(params[:id])
-    render json: game, status: 200
-  end
-
-  private
-  def param_validation
-    if !params[:id].present?
-      render json: { data: {} }, status:400
+    if !game.nil?
```

Comment:
use `game.present?`

Reply:
It is always nice to try and avoid inverse conditions when you can. Its not as simple to understand plus rails has cool stuff like `.present?` that developers are super familiar with.